bindings/protocols/coap/index.html

```diff
@@ -54,86 +54,280 @@ <h2>Introduction</h2>
+            </p>
+            <p>
+                The ... can be used to provide a list of one or more content formats that the Thing is able to provide in a CoAP response for the operation.
+                The Consumer selects the most suitable one and sets the <code>Accept</code> option in the request accordingly.
+            </p>
+            <pre id="example-request-accept" class="example" title="...">
+{
+    ...
+}
+</pre>
+            <p>
+                The ... can be used to provide a list of one or more content formats that the Thing is willing to accept in a CoAP request for the operation.
+                The Consumer selects the most suitable one and sets the <code>Content-Format</code> option in the request accordingly.
+            </p>
+            <pre id="example-request-format" class="example" title="...">
+{
+    ...
+}
+</pre>
+        </section>
+        <section>
+            <h3>Caching</h3>
+            <p>
+                CoAP features two mechanisms to make efficient use of network resources and minimize the number of required transmissions.
+                First, a CoAP client may store responses to requests in order to satisfy future requests without sending a new request.
+                A stored response may be used if it is still valid, i.e., if the response's Max-Age value has not been exceeded.
+                Second, a CoAP client may validate a stored response by sending a new request with the ETag from the stored response.
+                This allows the server to quickly respond with a 2.03 (Valid) response if the stored response is still valid.
+            </p>
+            <p>
+                Consumers are generally is expected to make use of both mechanisms to minimize the number of required CoAP transmissions.
+            </p>
+        </section>
+        <section>
+            <h3>Observing Resources</h3>
+            <p>
+                CoAP provides a mechanism for clients to "observe" resources [[RFC7641]], i.e., to request notifications from the server whenever the resource changes.
+                This is especially useful in the Web of Things where many Things are expected to have resources that change over time, such as sensor readings.
+            </p>
+            <p>
+                A resource can be observed by sending a GET request with the Observe option set.
+                The server will then respond with the current representation of the resource, as well as send notifications to the client.
+                A server is not required to support the Observe option for all resources; in this case, a resource can still be observed by polling it at regular intervals.
+            </p>
+            <p>
+                Observing resources can be enabled by including a <code>subprotocol</code> value of <code>cov:observe</code> in a form.
```

Comment:
How about this?

```suggestion
                Observable CoAP resources can be described by including a <code>subprotocol</code> value of <code>cov:observe</code> in a form.
```

Reply:
There’s not really something like an “observable resource”. Either a client wants a current representation of a (GET-able) resource, then it can observe the resource, or it doesn’t. RFC7641 is not required or something that needs to be negotiated for this; a client can always just poll the resource if it wants such a representation. Using RFC7641 just makes this slightly more efficient. So I’m not sure what’s best to write here…

Maybe if you have a “dumb” client that’s unsure if it should have a current representation over a period of time or not (such as a dashboard displaying property values), the TD could provide a hint to the client to indicate whether to keep the displayed value automatically up-to-date or to fetch it one time.

Reply:
I agree with @ektrah comment above. What I meant is that if we have `observeproperty` in the TD, it is not that suddenly the Thing implementation changes and the resource becomes observable (not sure what is the correct terminology).

Reply:
Hmm, would a better formulation be something like "Including a `subprotocol` value of `cov:observe` in a form indicates that the resource supports observation as specified by RFC 7641"? Or do you have a different suggestion?

Reply:
I've updated the text along those lines.